cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/BuildCommand.java

```diff
@@ -280,7 +280,7 @@ public void execute() {
             //// output flag cannot be set for projects
             if (null != this.output) {
                 CommandUtil.printError(this.errStream,
-                        "'-o' and '--output' flag is only supported for building a single ballerina file.",
+                        "the '-o' or '--output' flags are supported only when building a standalone Ballerina file.",
```

Comment:
Should this be "flag" or "flags". In https://github.com/ballerina-platform/ballerina-lang/pull/17596/files#diff-b4885eb61207cdeb93c14da32db64bf2R233 it says flag.

Reply:
Changed as follows:

"the '-o' or '--output' flag is supported only when building a standalone Ballerina file."